src/networkAddresses.ts

```diff
@@ -65,8 +65,8 @@ export const NETWORK_ADDRESSES = {
     REDSTONE_ORACLE: "0x4e6269Ef406B4CEE6e67BA5B5197c2FfD15099AE",
     MOCK_CRV: "0x2c78EF7eab67A6e0C9cAa6f2821929351bdDF3d3",
     MOCK_crvUSD: "0x36421d873abCa3E2bE6BB3c819C0CF26374F63b6",
-    XVS_VAULT_PROXY: sepolia_deployed_contracts.contracts.XVSVaultProxy.address,
-    XVS_STORE: sepolia_deployed_contracts.contracts.XVSStore.address,
+    XVS_VAULT_PROXY: "0x1129f882eAa912aE6D4f6D445b2E2b1eCbA99fd5",
+    XVS_STORE: "0x03B868C7858F50900fecE4eBc851199e957b5d3D",
```

Comment:
We can/should remove this entry from this file. We are not using `XVS_STORE` in the VIP framework. The VIP's should set their own address for this contract

Reply:
[10c9e12](https://github.com/VenusProtocol/vips/pull/143/commits/10c9e125d11fe89338de6f857c1fc1fadb36a402)